src/the-bureau/about-director/index.html

```diff
@@ -15,16 +15,16 @@
 
 	{% import 'macros/share.html' as share %}
 
-    <div class="block block__flush-top">
-        <h1>
+    <div class="block block__flush-top" data-qa-hook="director-bio">
+        <h1 data-qa-hook="director-bio-title">
             Director Richard Cordray
         </h1>
-        <p class="h3">
+        <p class="h3" data-qa-hook="director-bio-summary">
             Richard Cordray serves as the first Director of the
             Consumer Financial Protection Bureau. He previously
             led the Bureau’s Enforcement Division.
         </p>
-        <figure class="media float-media rwd-crop">
+        <figure class="media float-media rwd-crop"  data-qa-hook="director-bio-image">
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/8555b964c486de0b03084f0e58e3f97f44f7727a